libs/core/execution/tests/unit/algorithm_as_sender.cpp

```diff
@@ -30,7 +30,11 @@ int hpx_main()
         auto s = ex::as_sender(std::move(fut));
 
         static_assert(ex::is_sender_v<decltype(s)>);
+#ifdef HPX_HAVE_STDEXEC
+        static_assert(ex::is_sender_in_v<decltype(s), ex::empty_env>);
+#else
         static_assert(ex::is_sender_v<decltype(s), ex::empty_env>);
+#endif
```

Comment:
Shouldn't we rather implement `is_sender_in_v` for our code base?

Reply:
right, leaving that as a TODO to remove much of the duplication from the tests.